### PR TITLE
Dependabot security alert fixes

### PR DESCRIPTION
**Pull request by arnav28**

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -42,6 +42,10 @@
     "inquirer": "^8.1.0",
     "prettier": "^2.3.0"
   },
+  "resolutions": {
+    "trim": "^0.0.3",
+    "node-fetch": "^2.6.1"
+  },
```
Comment (jescalan) [inline website/package.json:48]: I'm not sure this is the way we'd want to go about fixing these - this is a fairly unusual fix. Is there any way we could leave this one out of this specific patch?

**Review by jescalan — CHANGES_REQUESTED** [review]

**Review by hashishaw — APPROVED** [review]

File: website/package.json (hunk 1 above)
Comment (hashishaw) [reply website/package.json:48]: Out of curiosity, how do you go about fixing these when they aren't direct dependencies? Our resolutions section is getting fairly large and I'd love to find a different way to address this

Comment (arnav28) [reply website/package.json:48]: Sure, I will remove it from the website for now and we can discuss it offline. Thanks :)

Comment (jescalan) [reply website/package.json]: Generally, we try to update the dependency and/or get an update merged upstream wherever possible 😀

**Review by jescalan — APPROVED** [review]

**Pull request merged**